src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -174,8 +213,26 @@ export default function ResearcherInfo(props) {
             type: 'button', // default button element type inside a form is "submit".
             className: 'button button-white btn-xs',
             style: { marginTop: 25 },
-            onClick: () => {}
-          }, ['Add Collaborator'])
+            onClick: () => addNewCollaborator(),
+            onChange: ({key: name, value,}) => formFieldChange({name, value}),
+          }, [
+            span({}, ['Add Collaborator'])
+          ]),
+
+          collaboratorsState
```

Comment:
I'm not following what the purpose of `collaboratorsState` is. The list of lab staff is stored in `labCollaborators`, which is being sent in to this component as a prop. Why not use that value directly?